### PR TITLE
[FEAT] : Quizzes page

### PR DESCRIPTION
**Pull request by VedanthB**
- added the quiz-page
- the preview like is https://deploy-preview-2--pensive-mcclintock-1405fd.netlify.app/pages/quiz-page.html

### CODE REVIEW

File: index.css [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+}
+
+.search__icon {
+  position: relative;
+  left: -30px;
+}
+/* header search bar */
+
+.hero-heading {
+  background: #e879f9;
```
Comment (Ananyamadhu08) [inline index.css:47]: use variables here

**Review by Ananyamadhu08 — APPROVED** [review]
Looks good to go 🚀

File: index.css (hunk 1 above)
Comment (VedanthB) [reply index.css:47]: Okay will make the changes

File: index.css [hunk 2]
```diff
@@ -0,0 +1,68 @@
+@import url("https://argon-css.netlify.app/components/index.css");
+
+/* header search bar */
+
+.search__bar__container {
+  width: 30rem;
+  display: flex;
+  justify-content: center;
+  align-items: center;
```
Comment (sharathnair9999) [inline index.css:9]: As making div center is a common use case you can use a combined utility class. Totally a suggestion. Not an improvement.

**Review by sharathnair9999 — APPROVED** [review]

**Review by sharathnair9999 — APPROVED** [review]

**Review by sharathnair9999 — APPROVED** [review]

**Review by sharathnair9999 — APPROVED** [review]

File: index.css (hunk 2 above)
Comment (VedanthB) [reply index.css:9]: okay, i already have a clasd for that, will use it here

**Pull request merged**